config/locales/models/activity.en.yml

```diff
@@ -110,6 +110,9 @@ en:
         fstc_applies:
           "true": "Yes (1)"
           "false": "No (0)"
+        is_oda:
+          "true": "ODA (1)"
```

Comment:
Is there anywhere else we display the numeric values other than `fstc_applies`? I can see it might be useful to associate the numbers with the bulk upload, but without that context, I wonder if it might be confusing to users? Maybe this is something we can check with BEIS or test as a post-MVP iteration.

Reply:
Does this mean "ok, keep", or "remove from current PR"?